package.json

```diff
@@ -127,12 +61,111 @@
             ]
           }
         ]
-      }
+      },
+      "makers": [
+        {
+          "name": "@electron-forge/maker-zip",
+          "platforms": [
+            "darwin"
+          ]
+        },
+        {
+          "name": "@electron-forge/maker-dmg",
+          "config": {
+            "format": "ULFO"
+          }
+        }
+      ],
+      "plugins": [
+        [
+          "@electron-forge/plugin-webpack",
+          {
+            "mainConfig": "./webpack.main.config.js",
+            "renderer": {
+              "config": "./webpack.renderer.config.js",
+              "entryPoints": [
+                {
+                  "html": "./src/renderer/index.html",
+                  "js": "./src/renderer/index.tsx",
+                  "name": "main_window"
+                }
+              ]
+            }
+          }
+        ]
+      ]
     }
   },
+  "dependencies": {
+    "@artossystems/a": "^3.1.0",
+    "electron-squirrel-startup": "^1.0.0",
+    "electron-store": "^3.2.0",
+    "execa": "^1.0.0",
+    "immer": "^3.1.2",
+    "jsonpath": "^1.0.2",
+    "lodash": "^4.17.11",
+    "mousetrap": "^1.6.3",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "styled-components": "^4.3.1",
+    "styled-system": "^4.0.0",
+    "update-electron-app": "^1.3.0",
+    "xml2js": "^0.4.19"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.4.5",
+    "@babel/preset-env": "^7.4.5",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.3.3",
+    "@electron-forge/cli": "6.0.0-beta.39",
+    "@electron-forge/maker-deb": "6.0.0-beta.39",
+    "@electron-forge/maker-dmg": "^6.0.0-beta.39",
+    "@electron-forge/maker-rpm": "6.0.0-beta.39",
+    "@electron-forge/maker-squirrel": "6.0.0-beta.39",
+    "@electron-forge/maker-zip": "6.0.0-beta.39",
+    "@electron-forge/plugin-webpack": "6.0.0-beta.39",
+    "@marshallofsound/webpack-asset-relocator-loader": "^0.5.0",
+    "@testing-library/react": "^8.0.1",
+    "@types/jest": "^24.0.13",
+    "@types/jsonpath": "^0.2.0",
+    "@types/lodash": "^4.14.133",
+    "@types/mousetrap": "^1.6.2",
+    "@types/node": "^12.0.4",
+    "@types/react": "^16.8.19",
+    "@types/react-dom": "^16.8.4",
+    "@types/styled-components": "^4.1.16",
+    "@types/styled-system": "^4.2.1",
+    "@types/xml2js": "^0.4.4",
+    "@typescript-eslint/eslint-plugin": "^1.9.0",
+    "@typescript-eslint/parser": "^1.9.0",
+    "@will-stone/prettier-config": "^1.0.0",
+    "babel-loader": "^8.0.6",
+    "browser-logos": "alrra/browser-logos#52.0.0",
```

Comment:
@will-stone Any reason(s) why you are not using the [`npm` packages](https://www.npmjs.com/org/browser-logos)?

Reply:
Hmmm, I can’t remember, I think it was because I couldn’t get them to import properly. Is there a reason to import them separately over this method? The way they are imported via web pack means the excess is tree shaken out.

Reply:
>  Is there a reason to import them separately over this method?

@will-stone Using this method has some disadvantages, including: 

* The structure of the repository on major releases may require [code changes](https://github.com/will-stone/browserosaurus/pull/105/files#diff-f337ad9cd1f802a63d3680ac8d16d045R5) (`src/` can become `src/archive/` or something else). This is not an issue with the `npm` packages as they contain just the images.
* Not getting patch and minor releases automatically (presuming `^<version>` is used).
* Slower install as the whole project with all the logos is downloaded.

Reply:
Okay, yeah that does seem valid. I'm not really sure what was failing before but maybe it was before I'd moved to webpack. I'll give it another go later. Thanks.

Reply:
I've used the `@browser-logos` versions, it's been pushed to `master`. I've used svg where possible, the only logo I needed that was missing is for the Min browser. I've just copied that one in so that I could remove the "browser-logos" package. It seemed to shave off about a mb from 63mb so nothing to write home about there, but at least the setup install will be much smaller. Thanks.

Reply:
👍🏻